docs/models.yml

```diff
@@ -515,7 +525,7 @@ meeting:
     default: outside
   motions_line_length:
     type: number
-    minimium: 40
+    minimum: 40
```

Comment:
Shouldn't this have been caught by the validator?

Reply:
I thought so.....

I'll have a look later and also add template fields